app/models/cohort.rb

```diff
@@ -1,51 +1,64 @@
 # frozen_string_literal: true
 
 class Cohort < ApplicationRecord
-  has_many :schedules, class_name: "Finance::Schedule"
-  has_many :partnerships
-  has_many :statements
   has_many :call_off_contracts
   has_many :npq_contracts
+  has_many :partnerships
+  has_many :schedules, class_name: "Finance::Schedule"
+  has_many :statements
+
+  # Class Methods
+
+  # Find the cohort with start year matching the given year.
+  def self.[](year)
```

Comment:
Im not too sure about exposing this new interface for lookups, my preference would be to still use `find_by(start_year:)` or the current/next/previous methods as I prefer the explicitness. But happy to hear the team's thoughts on this. 

Or maybe for 2020 because it's a special year we have scopes instead, let me know your thoughts

Reply:
This is just syntax sugar to shorten typing when you need to find a cohort by start year. 

It is so massively used everywhere in the test suite that I opted for:
 `Cohort[2021] || create(:cohort, start_year: 2021)` 
instead of
 `Cohort.find_by_start_year(2021) || create(:cohort, start_year: 2021)` 

Happy to revert it if people don't find it useful.

Reply:
Yes,  I agree with @cwrw

Reply:
reverted